modules/system/console/MixList.php

```diff
@@ -54,9 +50,15 @@ public function handle(): int
             }
         }
 
-        $this->table(['Name', 'Active', 'Path', 'Configuration'], $rows);
-
-        $this->line('');
+        if($this->option('json')) {
+            echo json_encode($rows, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);
```

Comment:
This should still go through the Laravel output methods for compatibility.

```suggestion
        if ($this->option('json')) {
            $this->line(json_encode($rows, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES));
```